src/board/alu.ts

```diff
@@ -56,7 +56,7 @@ export function sub(w1: Word, w2: Word): AluResult {
   flags[Flag.C] =
     w1.toUnsignedInteger() + comp.toUnsignedInteger() > Word.MAX_VALUE
   flags[Flag.V] =
-    w1.hasSign() != w2.hasSign() && w1.hasSign() != result.hasSign()
+    w1.hasSign() != w2.hasSign() && w1.hasSign() !== result.hasSign()
```

Comment:
Why only with result.hasSign() and not with w.2hasSign() as well?

Reply:
yes you are right I missed that one. Will change that.

Reply:
Changed